### PR TITLE
Re-order of Guides

### PR DESCRIPTION
**Pull request by benarent**
This PR re-orders a few sections of guides into more relevant sections. Mainly breaking out infrastructure from SSO guides. 

Screenshot has been reduced to make it clearer for a review. 

image

### CODE REVIEW

**Comment by klizhentas** [comment]
retest this

**Comment by klizhentas** [comment]
retest this

File: docs/4.1/installation.md [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
-This page will show you how to install Teleport. Teleport core services and
-admin tools [`tctl`] has been designed to run on **Linux** and **Mac** operating
-systems. The Teleport client [`tsh`] and Web UI are available for **Linux, Mac**
-and **Windows** operating systems. Our examples install teleport v4.1.0 but you
-can install any version listed in our [Release
-History](https://gravitational.com/teleport/releases/).
+Teleport core service [`teleport`](./cli-docs/#teleport) and admin tools [`tctl`](./cli-docs/#tctl) has been designed to run on **Linux** and **Mac** operating systems. 
+
+The Teleport client [`tsh`](./cli-docs/#tsh) and Web UI are available for **Linux, Mac**
+and **Windows** operating systems. Our examples install teleport v4.1.0 but you can
```
Comment (webvictim) [inline docs/4.1/installation.md]: `Teleport` (capitalisation)

File: docs/4.1/installation.md [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
-admin tools [`tctl`] has been designed to run on **Linux** and **Mac** operating
-systems. The Teleport client [`tsh`] and Web UI are available for **Linux, Mac**
-and **Windows** operating systems. Our examples install teleport v4.1.0 but you
-can install any version listed in our [Release
-History](https://gravitational.com/teleport/releases/).
+Teleport core service [`teleport`](./cli-docs/#teleport) and admin tools [`tctl`](./cli-docs/#tctl) has been designed to run on **Linux** and **Mac** operating systems. 
+
+The Teleport client [`tsh`](./cli-docs/#tsh) and Web UI are available for **Linux, Mac**
+and **Windows** operating systems. Our examples install teleport v4.1.0 but you can 
+install  any version listed in our [Release History](https://gravitational.com/teleport/releases/).
```
Comment (webvictim) [inline docs/4.1/installation.md]: `install any` (double space)

**Review by webvictim — APPROVED** [review]
LGTM with a couple of nits

**Comment by benarent** [comment]
@klizhentas , I see you re-testing this. The docs seem to still fail Jenkins / Force testing? Is it possible to run a different test suite for the `docs` directory?  We could then check for broken links and other static page issues?

**Pull request merged**